### PR TITLE
Restyled extensions

### PR DESCRIPTION
**Pull request by xsebek**
Just trying this out...

### CODE REVIEW

**Comment by xsebek** [comment]
I looked at https://github.com/project-chip/connectedhomeip/blob/master/.restyled.yaml, this should work. :confused:

**Comment by xsebek** [comment]
Here is the relevant wiki:
https://github.com/restyled-io/restyled.io/wiki/Configuring-Restyled#restylers-1

File: .hlint.yaml [hunk 1]
```diff
@@ -9,7 +9,26 @@
 # Specify additional command line arguments
 #
 # - arguments: [--color, --cpp-simple, -XQuasiQuotes]
-
+- arguments:
+  - '-XBangPatterns'
```
Comment (TristanCacqueray) [inline .hlint.yaml:13]: I think this should be: `--ghc-opt -XBangPatterns --ghc-opt -XDeriveAnyClass ...`

Comment (xsebek) [reply .hlint.yaml:13]: You probably mean a different file, but thanks, it helped me find the right thing. :+1:

The syntax I needed was ['--ghc-opt=-X...','--ghc-opt=-X',...]

**Pull request closed without merging**